src/diff/props.js

```diff
@@ -118,7 +118,8 @@ export function setProperty(dom, name, value, oldValue, isSvg) {
 			name in dom
 		) {
 			try {
```

Comment:
On a side note, this is a bug fix. I noticed this in my debugging sessions. For whatever reason, preact is setting the class name in dom even though it is identical and that causes a layout shift. This fixes it.

Reply:
It looks like this could be extracted into a seperate PR so the bug fix isn't blocked by the rest of this PR?

Reply:
ideally, yes. I'm hoping this PR will not take a long time to merge and it's a small bug that does not require immediate fix.